pynucastro/rates/library.py

```diff
@@ -608,9 +604,9 @@ def invert(self):
 
 class ReacLibLibrary(Library):
 
-    def __init__(self, libfile='reaclib_default2_20220329', rates=None, read_library=True):
-        assert libfile == 'reaclib_default2_20220329' and rates is None and read_library, "Only the reaclib_default2_20220329 default ReacLib snapshot is accepted"
-        Library.__init__(self, libfile=libfile, rates=rates, read_library=read_library)
+    def __init__(self, libfile='reaclib_default2_20220329'):
+        libfile == 'reaclib_default2_20220329'
```

Comment:
missing assert?

Reply:
my theory was that the assert is not necessary.  Oh ... I see.  I actually wanted to remove libfile as an arg and just set it manually, so this class will always just load the latest ReacLib.  I'll fix that.

Reply:
Yeah, I think removing the argument altogether makes the most sense

Reply:
updated